### DISCUSSION TITLE
android_wear_micro_apk.apk missing in /res

### DISCUSSION THREAD

**Discussion by oakeley** [NightscoutFoundation/xDrip#3528] [Bug reports]
Hi
The nightly builds don't seem to contain the android_wear_micro_apk.apk file in the /res folder any more. I unpacked and searched the entire nightly apk from 1st and 11th June and the file is no longer included. Would it be possible for you to add it back or update the documentation to say where it can now be found. Many thanks
![image](https://github.com/NightscoutFoundation/xDrip/assets/228558/a171df6f-de48-4093-99ae-a45b85871d9e)

**Comment by jonprogers** [comment]
I don't use a wear device so have no direct experience although I remembered this previous discussion. Don't know if it will help you...
https://github.com/NightscoutFoundation/xDrip/discussions/3261#discussioncomment-7983726

**Reply by oakeley** [reply]
Thanks, that link doesn't really provide a solution and part of my request is that someone updates the documentation to say where the apk can be found and how it can be extracted. Rather than compile apktool, install on phone, extract the wear.apk on the phone, transfer to computer and adb install on the watch... it is quicker to just compile xDrip and pull the wear apk from the build folder in: xDrip-20YY.MM.DD/app/build/generated/res/microapk/prod/release/raw
![image](https://github.com/NightscoutFoundation/xDrip/assets/228558/190dabb8-3abe-4f9d-80bb-b425fc1725fe)

**Reply by PhilTU** [reply]
I found that the file named sq.apk in the /res folder is the android_wear_micro_apk.apk package, I renamed it and installed it via adb. The rename may have been redundant.
